src/tests/utils/ApiTestSteps.ts

```diff
@@ -301,10 +301,15 @@ export function validateJwtToken(jwtToken: any): void {
 	validateRawBody(rawBody);
 }
 
-function validateRawHead(rawHead: any): void {
+async function validateRawHead(rawHead: any): Promise<void> {
```

Comment:
Since `validateRawHead` has been changed to `async` do we now need to `await` it where it's called to make sure the test fully executes?

Reply:
we have an await for `const hashBuffer = await crypto.subtle.digest("SHA-256", msgBuffer);`

Reply:
we have an await on `validateRawHead` just a few lines above